sdk/communication/communication-job-router/review/communication-job-router.api.md

```diff
@@ -128,9 +128,11 @@ export interface CreateJobOptions extends JobRouterUpsertJobOptionalParams {
     classificationPolicyId?: string;
     dispositionCode?: string;
     labels?: JSONObject;
+    // Warning: (ae-forgotten-export) The symbol "JobMatchingMode" needs to be exported by the entry point index.d.ts
```

Comment:
I think you need to export this JobMatchingMode type to get rid of this warning

Reply:
I'm not sure we want that? Any of these instances where we import a generated type to rename the interface or retype a property generate this warning. Do we want to export both our 'refined' type as well as the 'crude' generated type?

Reply:
In this case the interface RouterJobMatchingMode extends JobMatchingMode.  So we do want to export the base one, otherwise they don't know what it's extending.

Reply:
Ok, all warnings gone.